code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -565,7 +565,7 @@ get_true_breath_pressure(pp) --> gas_pp = pp/breath_pp*total_moles()
 		var/transfer_moles = (pressure_delta*output_air.volume)/(temperature * R_IDEAL_GAS_EQUATION)
 
 		//Actually transfer the gas
-		var/datum/gas_mixture/removed = remove(transfer_moles)
+		var/datum/gas_mixture/removed = remove(transfer_moles * rate)
```

Comment:
This breaks everything that doesn't pass in a rate, and is also kinda stinky I think. 
Do you need this exact behavior? Would modulating the target pressure work alright?

Reply:
Oh woops yeah I forgot to give it a default value. Modulating pressure is very unwieldy for this which is why I switched away from using it.

Reply:
I still really don't like how this smells, but it's functional

Reply:
I'm still not sure how it's a smell to have an explicit rate argument for a proc moving gas around

Reply:
Well the idea of the proc is to release enough gas from mix one into mix 2 to reach the target pressure, and scaling the moles moved kinda stinks

Reply:
not much to do about it tho